pkg/controller/registry/reconciler/reconciler_test.go

```diff
@@ -74,3 +74,20 @@ func TestPullPolicy(t *testing.T) {
 		}
 	}
 }
+
+func TestPodRunAsUser(t *testing.T) {
+	source := &v1alpha1.CatalogSource{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "test",
+			Namespace: "test-ns",
+		},
+	}
+
+	p := Pod(source, "hello", "busybox", "", map[string]string{}, map[string]string{}, int32(0), int32(0))
+	user := p.Spec.SecurityContext.RunAsUser
+	if user == nil {
+		t.Fatal("expected RunAsUser value for pod expected to not be nil")
+	} else if *user != int64(1001) {
+		t.Fatalf("expected RunAsUser value for pod expected to be %v, not %v", int64(1001), *user)
+	}
```

Comment:
Minor nit, but I think we can use the `require` package here to validate the Pod matches our expectations:

```go
require.NotNil(t, user, "expected RunAsUser ...")
require.EqualValues(t, ...)
```

Reply:
Should be updated in https://github.com/operator-framework/operator-lifecycle-manager/pull/2023/commits/60e36c0c998b77c40fa6af6759eca4bdeac478e6